handlers/desired_lrp_handlers.go

```diff
@@ -90,8 +95,8 @@ func (h *DesiredLRPHandler) commonDesiredLRPs(logger lager.Logger, targetVersion
 	}
 
 	response.Error = models.ConvertError(err)
-	writeResponse(w, response)
-	exitIfUnrecoverable(logger, h.exitChan, response.Error)
+	// writeResponse(w, response.ToProto())
+	// exitIfUnrecoverable(logger, h.exitChan, response.Error)
```

Comment:
These lines should probably get removed

Reply:
Cleaned up in [f7314a9](https://github.com/cloudfoundry/bbs/pull/109/commits/f7314a95de15266d952902af8767dd6b5a741df3)